.gitlab-ci.yml

```diff
@@ -473,8 +473,9 @@ Installer:
     matrix:
       - RUNNER:
           - rhos-01/rhel-8.6-nightly-x86_64
-          - rhos-01/rhel-9.0-nightly-x86_64
-          - rhos-01/centos-stream-9-x86_64
+          # disable until https://bugzilla.redhat.com/show_bug.cgi?id=2059565 is fixed
+          #- rhos-01/rhel-9.0-nightly-x86_64
+          #- rhos-01/centos-stream-9-x86_64
```

Comment:
@thozza can you open a ticket in Jira with the `[QE]` prefix (and a relatively high priority) so that we don't forget to enable this?

Reply:
Will do so tomorrow...